ArduPlane/servos.cpp

```diff
@@ -968,6 +968,23 @@ void Plane::landing_neutral_control_surface_servos(void)
  
 }
 
+/*
+  sets rudder/vtail , and elevon to indicator positions that we are in a rudder arming waiting for neutral stick state
+*/
+void Plane::indicate_waiting_for_rud_neutral_to_takeoff(void)
+{
+    if (waiting_for_rudder_neutral)  {
```

Comment:
I think we need to reset waiting_for_rudder_neutral on mode change, and also have a FLIGHT_OPTIONS bit to enable this behaviour